addons/website_crm_partner_assign/models/crm_lead.py

```diff
@@ -218,7 +218,7 @@ def update_lead_portal(self, values):
         for lead in self:
             lead_values = {
                 'planned_revenue': values['planned_revenue'],
-                'probability': values['probability'],
+                'probability': values['probability'] or False,
```

Comment:
I was about to suggest values.get('probability', False) but then I saw the 'date_deadline' here below soooo heh, it's fine :D

Reply:
It will not work because the dict contain `None` as value (the key is set)
`{'probability': None}`, so if we "get" the value we will have None and not False :/